godot/game/game.gd

```diff
@@ -67,12 +67,20 @@ func _process(delta):
 		if death_timer.is_stopped():
 			sound.play()
 		sec_since_last_magician_sfx = -sound.stream.get_length()
-
-	var r = randi()%(450+int(game_timer.time_left*1.5))
+	
+	environment.set_game_progress_ratio(1.-game_timer.time_left/max_game_time)
+	
+	# spawn objects
+	#var r = randi()%(450+int(game_timer.time_left*1.5))
+	var masterProbability = 450.+game_timer.time_left*1.5
+	masterProbability = masterProbability*(1./60.)/delta
```

Comment:
Benennung der Variablen sollten den Coding conventions folgen. Bitte schau dir nochmal an wie die restlichen Variablen benannt sind

Reply:
Schreib bitte auch mal noch einen erklärenden Kommentar zu diesen Wahrscheinlichkeitsberechnungen. Ich verstehe glaube ich so grob was hier passiert, aber in 'ner Woche wird das keiner mehr checken.

Reply:
Die Wahrscheinlichkeiten wären besser als Brüche (%) pro Sekunde die man dann einfach mit delta multipliziert um auf die Frameabhängige Wahrscheinlichkeit zu kommen, ich würde es vorerst als int lassen, damit es erstmal genau so schwer ist wie intended.